nncf/dynamic_graph/patch_pytorch.py

```diff
@@ -204,3 +204,30 @@ def unpatch_torch_operators():
 
     for orig_op_info in ORIGINAL_OPERATORS:
         setattr(orig_op_info.namespace, orig_op_info.name, orig_op_info.op)
+
+
+def patch_extension_build_function():
+    """
+    The function patches PyTorch and fix a bug inside CUDA extensions building;
+    The bug must be fixed with a new PyTorch 1.8.0
+    """
+    import torch.utils.cpp_extension
+
+    if torch.__version__ == '1.8.0':
```

Comment:
Should check for version larger than 1.8.0 instead of checking version equality.

Reply:
Torch developers could change function name or somehow change it, so I think that it is better to keep the versions in which we are sure

Reply:
[Semantic versioning](https://www.python.org/dev/peps/pep-0440/) in Python packages is a well-defined concept, and once a project starts using it, it rarely deviates from it. Expecting a format change here is unreasonable.

Reply:
It is more likely that during the next update we will transition directly to a torch version 1.8.1 should it be available at that moment, and your code may break.

Reply:
Updated

Reply:
Reupdated